jaxrs-tck/pom.xml

```diff
@@ -40,6 +40,14 @@
             <artifactId>jakarta.ws.rs-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
<scope>provided</scope>
            <optional>true</optional>

Reply:
FWIW I don't think it should be `<optional>true</optional>` as it's not optional, but should be provided by the user using the TCK.

Reply:
Right, my bad, it's the TCK, not the API.